trailmet/algorithms/prune/network_slimming_train.py

```diff
@@ -0,0 +1,253 @@
+from __future__ import print_function
```

Comment:
i am skipping review of this for now. There should be a singile file with organized content.